### PR TITLE
add anakin softmax/transpose/batch_norm/flatten/reshape op

### PR DESCRIPTION
**Pull request by fc500110**

### CODE REVIEW

File: paddle/fluid/inference/anakin/convert/ut_helper.h [hunk 1]
```diff
@@ -177,6 +178,9 @@ class AnakinConvertValidation {
       auto fluid_out = fluid_outputs[i_output++];
       for (size_t i = 0; i < anakin_out_size; i++) {
         EXPECT_LT(std::abs(fluid_out[i] - anakin_out[i]), 1e-3);
+        //LOG(INFO) << "Output[" << i << "]: anakin[" << anakin_out[i] << "], "
+        //          << "fluid[" << fluid_out[i] << "]";
```
Comment (NHZlX) [inline paddle/fluid/inference/anakin/convert/ut_helper.h]: 没用的注释删掉吧

**Review by Superjomn — APPROVED** [review]
LGTM
